pkg/controller/networkpolicy/controller_querier_replier.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2020 Antrea Authors
```

Comment:
The file name "controller_querier_replier" sounds a little uncommon, why don't just call it "endpoint_querier" given the interface name is EndpointQuerier?

Reply:
Fixed in https://github.com/vmware-tanzu/antrea/pull/1116